frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -36,6 +36,10 @@ export const SightingCard: React.FunctionComponent<SightingProps> = ({
           <span className="sighting-card-information">No. whales: </span>
           {sighting.whaleCount}
         </p>
+        <p>
+          <span className="sighting-card-information">Location: </span>
+          {sighting.location}
+        </p>
```

Comment:
Display conditionally on there being a location (can't guarantee it now)

Reply:
done, thanks